src/Pages/Platforms/forecastType.tsx

```diff
@@ -6,13 +6,15 @@ import { RouteComponentProps } from "react-router-dom"
 
 import { Forecast } from "Features/ERDDAP"
 import { RenderProps } from "Features/ERDDAP/Platform/Grabber"
+import { useUnitSystem } from "Features/Units"
 
 import { PlatformObservationMatchParams } from "./types"
 
 export type ForecastTypePageProps = RenderProps & RouteComponentProps
 
-export const ForecastTypePage: React.SFC<ForecastTypePageProps> = ({ platform, match, unit_system }) => {
+export const ForecastTypePage: React.SFC<ForecastTypePageProps> = ({ platform, match }) => {
   const { type } = match.params as PlatformObservationMatchParams
+  const unit_system = useUnitSystem()
```

Comment:
Codacy found an issue: [Identifier 'unit_system' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)